tikv-client/src/main/java/com/pingcap/tikv/TiDBJDBCClient.java

```diff
@@ -69,6 +77,70 @@ public boolean supportClusteredIndex() throws IOException, SQLException {
     return true;
   }
 
+  public List<String> showGrants() {
+    List<String> result = new ArrayList<>();
+    try (Statement tidbStmt = connection.createStatement()) {
```

Comment:
It should handle disconnected connection properly.

Reply:
I don't get it

Reply:
I should create a new jdbcClient class to handle this situation